core/trino-main/src/test/java/io/trino/type/TestIntegerOperators.java

```diff
@@ -213,7 +213,7 @@ public void testCastToVarchar()
         assertFunction("cast(INTEGER '17' as varchar)", VARCHAR, "17");
         assertFunction("cast(INTEGER '123' as varchar(3))", createVarcharType(3), "123");
         assertFunction("cast(INTEGER '123' as varchar(50))", createVarcharType(50), "123");
-        assertFunctionThrowsIncorrectly("cast(INTEGER '123' as varchar(2))", IllegalArgumentException.class, "Character count exceeds length limit 2.*");
```

Comment:
will be able to remove `assertFunctionThrowsIncorrectly` at the end?

Reply:
The casts from approximate numeric types to varchar are still to be fixed, and this method will be useful while doing this. I plan to take care of them soon. Afterwards, the method could be removed.